src/lib/bootloader/grub2base.rb

```diff
@@ -148,6 +150,27 @@ def disable_serial_console
 
   private
 
+    def system_locale
+      begin
+        language = ::Bootloader::Language.new
+        language.load
+      rescue Errno::ENOENT
```

Comment:
change is in https://github.com/config-files-api/config_files_api/pull/12

when playing with it, I found that exist? is not enough, as there can be problem with permissions, temporary read error and possible other problems.